colorclade/_biopython_draw_colorclade_tree.py

```diff
@@ -98,15 +109,17 @@ def biopython_draw_colorclade_tree(
             biopy_tree,
             axes=ax,
             label_func=lambda node: (
-                node.name if label_tips and node.is_terminal() else ""
+                node.name if label_tips(node) and node.is_terminal() else ""
```

Comment:
It would be a little more general to have label_tips take in a node and return the label, returning “” if the tip shouldn’t be labeled (this opens the door to users changing the label to be something other than name)